### PR TITLE
Python work

### PR DESCRIPTION
**Pull request by angrave**
Kaltura Playlist and Channel support
Quieter ffmpeg + timing
Reduced number of concurrent pythonrpc to a sensible number (production host has 8 cpu)

### CODE REVIEW

**Comment by angrave** [comment]
You can delete this branch once it is merged

File: PythonRpcServer/server.py [hunk 1]
```diff
@@ -76,11 +76,10 @@ def ProcessVideoRPC(self, request, context):
         filePath, ext = ffmpeg.processVideo(request.filePath)
         return ct_pb2.File(filePath = filePath, ext = ext)
 
-
 def serve():
     print("Python RPC Server Starting")
     
-    server = grpc.server(futures.ThreadPoolExecutor(max_workers=1000))
+    server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
```
Comment (robkooper) [inline PythonRpcServer/server.py:82]:
```suggestion
    server = grpc.server(futures.ThreadPoolExecutor(max_workers=int(os.getenv('NUM_THREADS', 10))))
```

**Review by robkooper — COMMENTED** [review]
lgtm, added small tweak to allow to set number of threads as an environment variable (probably need to import os).

File: PythonRpcServer/server.py (hunk 1 above)
Comment (angrave) [reply PythonRpcServer/server.py]: Added but with different env key 'NUM_PYTHON_WORKERS'

**Review by robkooper — APPROVED** [review]

**Pull request merged**